TopReadWidget/WMFTodayTopReadWidgetViewController.swift

```diff
@@ -348,18 +348,7 @@ class WMFTodayTopReadWidgetViewController: ExtensionViewController, NCWidgetProv
     }
 
     func fetch(dataStore: MWKDataStore, completionHandler: @escaping ((NCUpdateResult) -> Void)) {
-        updateUIWithTopRead(with: dataStore, moc: dataStore.viewContext) { result in
-            guard result == .failed else {
-                completionHandler(result)
-                return
-            }
-            dataStore.feedContentController.updateFeedSourcesUserInitiated(false) {
```

Comment:
I haven't tested to confirm but seems like there's a chance this could miss a daily update by stripping this network update. Did you purposefully not lean on your new `fetchNewestWidgetContentGroup(.topRead, in: dataStore, isNetworkFetchAllowed: true)` method here?

Reply:
Yeah, thanks for pointing this out, I should have mentioned this in the notes. The other legacy widgets don't perform a network fetch but instead rely on background updates for the app. I thought it might make more sense to be consistent with the other widgets, but it's probably better to stay consistent with the previous behavior of this widget. I restored the fetch using `fetchNewestWidgetContentGroup`.